components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -165,6 +167,9 @@
  */
 @property(nonatomic, strong, nullable) UIColor *snackbarMessageViewShadowColor;
 
+/** The elevation for the Snackbar message view. */
+@property(nonatomic, assign) MDCShadowElevation elevation;
```

Comment:
Nit: Naming: Please call this something like `messageElevation`. That will make it clearer the elevation is for the snackbar messages and not the manager (or something else).

Reply:
Done